partner_multi_relation_tabs/models/res_partner_relation_type.py

```diff
@@ -57,3 +57,14 @@ def _check_tab_right(self):
                     tab_partner_category_id != rec.partner_category_right:
                 raise ValidationError(_(
                     "Partner category right not compatible with right tab"))
+
```

Comment:
Leave out one blank line (functions within a class should be separated with a single blank line), and travis test will be green.

Reply:
Did that but Travis fails but it seems to me on something unrelated. I am quite newbie in hands-on experience with CI and stuff (git as well) so please bear with me. I also attempted to port this module as well at the additional one to work with Products - see here: https://github.com/OCA/product-attribute/pull/598 That for sure needs some work - I now found out what the tests directory is for so need to work on that there...